lib/challenge.js

```diff
@@ -60,6 +86,20 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function getPetInfo(name, species, breed, noise) = {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise
+  this.makeNoise = makeNoise();
+}
+
+var Pet = New getPetInfo();
```

Comment:
`new` is a keyword, and should not be capitalized. Also, although they're technically the same function in your code, the convention would be to use `Pet` rather than `getPetInfo`. 

Additionally, you shouldn't be assigning the result back into `Pet`. First, and most importantly, it overwrites your constructor function. And secondly, it goes against convention to capitalize anything but a constructor function; the result of `new` should not be another constructor function, but rather a new object.